ratatool-sampling/src/main/scala/com/spotify/ratatool/samplers/BigSampler.scala

```diff
@@ -122,6 +122,7 @@ object BigSampler extends Command {
         |  [--distribution=(uniform|stratified)]      An optional arg to sample for a stratified or uniform distribution. Must provide `distributionFields`
         |  [--distributionFields=<field1,field2,...>] An optional list of fields to sample for distribution. Must provide `distribution`
         |  [--exact]                                  An optional arg for higher precision distribution sampling.
+        |  [--byteHasher=(bytes|hex|base64)]          An optional arg for how to hash bytes to make the hash of a bytes field equal to a hex or baes64 represented string field.
```

Comment:
It's not clear what the default behaviour is here.

Reply:
Fixed